tiatoolbox/tools/stainnorm.py

```diff
@@ -149,7 +154,8 @@ class RuifrokNormalizer(StainNormalizer):
 
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
+        """Initializes :class:`RuifrokNormalizer`."""
```

Comment:
```suggestion
        """Initialize :class:`RuifrokNormalizer`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090